_data/software/anaconda.yml

```diff
@@ -0,0 +1,40 @@
+---
+schemaVersion: '1.0'
+description:
+  whatItDoes:
+    en: >-
+      Anaconda is a distribution of the Python and R programming languages for
+      scientific computing.
+    fr: >
+      Anaconda est une distribution des langages de programmation Python et R
+      pour le calcul scientifique.
+category: '43232400'
+homepageURL:
+  en: 'https://www.anaconda.com/'
+  fr: 'https://www.anaconda.com/'
+licences:
+  - URL:
+      en: 'https://docs.anaconda.com/anaconda/eula/'
```

Comment:
Is there a need to call out projects that have those extra caveats on their license? EX: licenses for the things they redistribute

Reply:
Not sure where we can place that right now, but in theory we'll want to expand on this in the future.